setup.py

```diff
@@ -3,30 +3,16 @@
 
 from setuptools import setup,find_packages
 
-import bypy
-doclist = bypy.__doc__.split("---")
-long_desc = doclist[1].strip() + '\n\n'
-try:
-	import pypandoc
-	long_desc += pypandoc.convert('HISTORY.md', 'rst')
-except Exception as ex:
-	print("Error: PanDoc not found\n{}".format(ex))
-	with open('HISTORY.md') as f:
-		long_desc += f.read()
-
-print(long_desc)
-
 setup(
 	name='bypy',
-	version=bypy.__version__,
-	description='Python client for Baidu Yun (Personal Cloud Storage) 百度云/百度网盘 Python 客户端',
-	long_description=long_desc,
+	version='1.2.7',
```

Comment:
Hi @aflyhorse 

For this one, I don't get it, why is referring to bypy.__version__ inappropriate?